cloud/linode/loadbalancers.go

```diff
@@ -572,14 +660,77 @@ func (l *loadbalancers) getNodeBalancerByID(ctx context.Context, service *v1.Ser
 }
 
 func (l *loadbalancers) getLoadBalancerTags(_ context.Context, clusterName string, service *v1.Service) []string {
-	tags := []string{clusterName}
+	tags := []string{}
+	if clusterName != "" {
+		tags = append(tags, clusterName)
+	}
+
 	tagStr, ok := service.GetAnnotations()[annLinodeLoadBalancerTags]
 	if ok {
 		return append(tags, strings.Split(tagStr, ",")...)
 	}
 	return tags
 }
 
+func processACL(fwcreateOpts *linodego.FirewallCreateOptions, aclType, label, svcName, ports string, ips linodego.NetworkAddresses) {
```

Comment:
this doesn't have docs + it's not quite clear from the name (or quite frankly from reading the code) what this is supposed to do (it only started becoming somewhat clear from context)